packages/app/src/components/presets/workos-docs.js

```diff
@@ -11,6 +11,30 @@ const query = {
 const code = (
   <Inline>
     <Box>
+      <style>
+        {`
+          @font-face {
+            font-family: 'Untitled Sans';
+            src: url('https://cdn.workos.com/www/open-graph/untitled-sans-regular.woff2') format('woff2');
+            font-weight: 400;
+            font-style: normal;
+            font-display: block;
+          }
+          @font-face {
+            font-family: 'Untitled Sans';
+            src: url('https://cdn.workos.com/www/open-graph/untitled-sans-medium.woff2') format('woff2');
+            font-weight: 500;
+            font-style: normal;
+            font-display: block;
+          }
+        `}
+      </style>
+
+      <Link
+        href="https://fonts.googleapis.com/css2?family=IBM+Plex+Mono:wght@500&display=block"
```

Comment:
I want to verify if the wrong font loading was actually the fact `cdn.workos` is not added as preconnect

Can we revert this for a while?

```suggestion
        href="https://fonts.googleapis.com/css2?family=IBM+Plex+Mono:wght@500&display=swap"
```

Reply:
Sure, updated.